site/source/docs/api_reference/emscripten.h.rst

```diff
@@ -1018,7 +1018,7 @@ Defines
 
 .. c:macro:: EM_LOG_ERROR
 
-  If specified, prints an error message. If neither :c:data:`EM_LOG_WARN`, :c:data:`EM_LOG_ERROR`, :c:data:`EM_LOG_INFO` nor :c:data:`EM_LOG_DEBUG` is specified, an info message is printed. :c:data:`EM_LOG_WARN`, :c:data:`EM_LOG_INFO`, :c:data:`EM_LOG_DEBUG` and :c:data:`EM_LOG_ERROR` are mutually exclusive.
+  If specified, prints an error message (combined with :c:data:`EM_LOG_CONSOLE`). If neither :c:data:`EM_LOG_WARN`, :c:data:`EM_LOG_ERROR`, :c:data:`EM_LOG_INFO` nor :c:data:`EM_LOG_DEBUG` is specified, a log message is printed. :c:data:`EM_LOG_WARN`, :c:data:`EM_LOG_INFO`, :c:data:`EM_LOG_DEBUG` and :c:data:`EM_LOG_ERROR` are mutually exclusive. :c:data:`EM_LOG_ERROR` combined with :c:data:`EM_LOG_WARN` will behave like :c:data:`EM_LOG_CONSOLE` combined with :c:data:`EM_LOG_ERROR`.
```

Comment:
This last sentence I don't think it correct.    I think its more like 

"If `EM_LOG_CONSOLE` is not specified than the message output via `err` (for EM_LOG_ERROR or EM_LOG_WARN) and `out`, otherwise."

(the line `else if (flags & 6 /*EM_LOG_ERROR|EM_LOG_WARN*/) {` is checking for either EM_LOG_ERROR *or* EM_LOG_WARN)